docs/tutorials/sinks/2_http_sink.md

```diff
@@ -98,16 +99,25 @@ sending batches of events, or they may send a completely different type if each
 event is processed in some way prior to encoding.
 
 [`encode_input`][encoder_encode_input] serializes the event to a String and
-writes these bytes:
+writes these bytes. The function also creates a
+[`GroupedCountByteSize`][grouped_count_byte_size] object. This object tracks the
+size of the event that is sent by the sink. This object can also track the size
+by the source and service that originated the event if Vector has been configured
```

Comment:
```suggestion
size of the event that is sent by the sink. This object can also track the size emitted
by the source and service that originated the event if Vector has been configured
```

Could be other ways, but it seemed to be missing a verb?

Reply:
I reworded it a bit, because that was a really awkward to read few sentences. Hopefully it makes more sense now?

Reply:
Nice, I like the revision :)